x/evm/statedb/journal.go

```diff
@@ -24,21 +24,21 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 )
 
-// journalEntry is a modification entry in the state change journal that can be
-// reverted on demand.
-type journalEntry interface {
-	// revert undoes the changes introduced by this journal entry.
-	revert(*StateDB)
+// JournalEntry is a modification entry in the state change journal that can be
+// Reverted on demand.
+type JournalEntry interface {
```

Comment:
Why did it need exporting? I can't see it used outside in this PR.

Reply:
Required for Stateful Precompiles down the line.